### DISCUSSION TITLE
Utilité du repo ?

### DISCUSSION THREAD

**Discussion by HubM** [bdxio/bdxio-yt-publisher#44] [Announcements]
Hello Benoit (@joow)  ! 
Je me demandais si ce repo était encore utilisé ? Et si on voulait le garder actif sur le github, ou si je l'archive ? 

Merci par avance pour ta réponse :) 

Hub

**Comment by joow** [comment]
Ce repo contient le code de l'outil qui permet de découper les streams live pour les publier sur YouTube.

Si vous ne comptez pas vous en resservir vous pouvez l'archiver.
